app/views/articles/_reaction_button.html.erb

```diff
@@ -1,6 +1,7 @@
 <button
-  role="checkbox"
+  type="button"
```

Comment:
```suggestion
```

Refactor (blocking) - no need for this line as it's already implied by the `<button>` element